src/Command/RunTemplateCommand.php

```
@@ -565,7 +574,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                     $isImmediate = ($scheduleDateTime->getTimestamp() <= $now->getTimestamp() + 5); // 5 sec tolerance
                     $scheduleType = $isImmediate ? 'adhoc' : 'cli';
 
-                    $prepared = $jobber->prepareJob($rt->getMyKey(), $overridedEnv, $scheduleDateTime, $executor, $scheduleType);
+                    $prepared = $jobber->prepareJob($rt, $overridedEnv, $scheduleDateTime, $executor, $scheduleType);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$prepared'. (undefined)

(UnusedLocalVariable)